src/Core/Framework/Store/Services/StoreClient.php

```diff
@@ -119,7 +124,11 @@ public function getExtensionUpdateList(ExtensionCollection $extensionCollection,
             ];
         }
 
-        return $this->getUpdateListFromStore($extensionList, $context);
+        return $this->cache->get(self::EXTENSION_LIST_CACHE, function (ItemInterface $item) use ($extensionList, $context) {
```

Comment:
Just curious, couldn't different users get different lists? Just in theory, do we block that different users are connected to different companies in the SBP?